generators/app/templates/public/locales/en/translation.json

```diff
@@ -9,7 +9,15 @@
   "notifications": {
     "success": "Success",
     "error": "Error",
-    "unknownError": "An unknown error has occurred"
+    "unknownError": "An unknown error has occurred",
+    "404": "Editing a new document.",
+    "errorLoading": "There was an error loading your file, please try again.",
+    "errorFetchin g": "Error fetching {{url}}.",
```

Comment:
They key here has an erroneous space in it (`errorFetchin g` instead of `errorFetching`)